notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,296 @@
+    "![AzureML Workflow](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/overview-what-is-azure-ml/aml.png)\n",
+    "\n",
+    "This noteboook provides a scaffold of how to directly submit an existing notebook to AzureML compute targets. \n",
+    "\n",
+    "### Advantages of using AzureML:\n",
+    "- Manage cloud resources for monitoring, logging, and organizing your machine learning experiments.\n",
+    "- Train models either locally or by using cloud resources, including GPU-accelerated model training.\n",
+    "- Easy to scale out when dataset grows - by just creating and pointing to new compute target\n",
+    "\n",
+    "### Prerequisities\n",
+    "   - **Azure Subscription**\n",
+    "       -  You can [create a free subscription](https://azure.microsoft.com/en-us/free/) or access existing subscription information from the Azure portal. Later in this notebook you will need information such as your subscription ID in order to create and access AzureML workspaces.\n",
+    "          - New subscription comes with [free credit](https://azure.microsoft.com/en-us/free/free-account-faq/) which will easily cover the cost of running this example notebook\n",
+    "\n",
+    "# Setup environment\n",
+    "\n",
+    "### Install azure.contrib.notebook package\n",
+    "Since azureml.contrib.notebook contains experimental components, it's not included in generated .yaml file by default."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install \"azureml.contrib.notebook>=1.0.21.1\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate reco_base.yaml file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Generated conda file: reco_base.yaml\r\n",
+      "\r\n",
+      "To create the conda environment:\r\n",
+      "$ conda env create -f reco_base.yaml\r\n",
+      "\r\n",
+      "To update the conda environment:\r\n",
+      "$ conda env update -f reco_base.yaml\r\n",
+      "\r\n",
+      "To register the conda environment in Jupyter:\r\n",
+      "$ conda activate reco_base\r\n",
+      "$ python -m ipykernel install --user --name reco_base --display-name \"Python (reco_base)\"\r\n",
+      "\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python ../../scripts/generate_conda_file.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "This notebook was created using version 1.0.21 of the Azure ML SDK\n",
```

Comment:
I took this from Azure/MachineLearningNotebooks, but I guess I should do as other notebooks in Recommender do. One question though, what if something doesn't run because os version upgrade? Is it obvious to user it maybe caused by version? I think line 

> You are currently using version 1.0.21 of the Azure ML SDK

 help with this.

Reply:
Changed to print("azureml.core version: {}".format(azureml.core.VERSION))